test/integration/targets/docker_swarm/tasks/tests/basic.yml

```diff
@@ -64,15 +64,15 @@
 - name: Create a Swarm cluster (force re-create)
   docker_swarm:
     state: present
-    advertise_addr: "{{ansible_default_ipv4.address}}"
+    advertise_addr: "{{ansible_default_ipv4.address | default('127.0.0.1')}}"
```

Comment:
I don't think a swarm can communicate via a local socket. Won't make much sense anyway, because you aren't supposed to have two swarm nodes on the same machine/VM.

Reply:
I was thinking wider than just integration tests tbh. Here it does not make sense, in real scenarios you just need to run the module on a host that is a manager.